docs/setup-robusta/alertsui.rst

```diff
@@ -0,0 +1,74 @@
+Alerts UI
+######################
+
+With Robusta Alerts UI, you can create and customize Prometheus Rules using templates, without needing to know PromQL.
+
+.. image:: /images/managed-alerts-ui.png
+
+
+
+This guide covers enabling managed alerts with Robusta UI. This involves two steps: One to enable the Alert Rules on the UI and another to disable default Prometheus alerts installed by Kube Prometheus Stack.
+
+.. note::
+
+    Alerts UI works only with Prometheus installed by Robusta. :ref:`Embedded Prometheus Stack`
+
+Enable Alerts UI
+--------------------
+
+To start seeing all the alerts and customize them, we first need to enable Alert Rules in the Robusta UI.
+
+1. Go to `platform.robusta.dev <https://platform.robusta.dev/>`_ -> Alerting -> Alerts
+2. Click "Enable Alert Rules"
+
+.. image:: /images/click-enable-alert-rules.png
+
+
+
+Enable Robusta Managed Alerts
```

Comment:
I'm a little confused by "Enable Alerts UI" vs "Enable Alert Rules" vs "Enable Robusta Managed Alerts"

They all sound like the same thing to me. Can we be more clear what each one is doing? 

(In your instructions, refer to the exact text on the buttons. E.g. you can write `Click the "Enable Alert Rules" button.` But in your titles and description of what is happening, try to make sense of it.)